_data/simple-icons.json

```
@@ -3188,6 +3178,11 @@
         },
         {
             "title": "DVC",
+            "aliases": {
+                "aka": [
+                    "DVC"
```

Shouldn't be "Data Version Control"?

```suggestion
                    "Data Version Control"
```